app/src/main/java/com/elmeradrianv/shesafe/fragments/MapFragment.java

```diff
@@ -66,12 +66,26 @@
 public class MapFragment extends Fragment implements OnMapReadyCallback, GoogleMap.OnMapLongClickListener {
     private static final String TAG = MapFragment.class.getSimpleName();
     private final static String KEY_LOCATION = "location";
-    private static final int UPDATE_INTERVAL = 60000; //In milliseconds, 60s
+    private static final int UPDATE_INTERVAL = 36000; //In milliseconds, 36s
+    private static final double TO_KM_PER_HOUR = 1000;
     private static final int FASTEST_INTERVAL = 5000; //In milliseconds, 5s
     private static final int SQUARE_GRID_LENGTH = 3;
     private static final int SQUARE_GRID_3X3_COUNT = SQUARE_GRID_LENGTH * SQUARE_GRID_LENGTH;
     private static final int SQUARE_CENTER_CENTER = 4;
     private static final int OUTSIDE_GRID = -1;
+    private static final double SPEED_MAX_WALK = 8;
+    private static final double SQUARE_SIZE_WALK = 0.003;
+    private static final double SPEED_MAX_BIKE = 20;
+    private static final double SQUARE_SIZE_BIKE = 0.009;
+    private static final double SQUARE_SIZE_CAR = 0.027;
+    private static final int MOV_WALK = 0;
```

Comment:
it might be good to make some of these into enums rather than just integers/doubles that way you can more specifically group them together by "type"

so using them would look like: Zoom.WALK rather than Zoom_Walk